statsmodels/graphics/gofplots.py

```diff
@@ -307,9 +357,21 @@ def qqplot(self, xlabel=None, ylabel=None, line=None, other=None,
             check_other = isinstance(other, ProbPlot)
             if not check_other:
                 other = ProbPlot(other)
-
-            fig, ax = _do_plot(other.sample_quantiles,
-                               self.sample_quantiles,
+            
+            s_self = self.sample_quantiles
+            s_other = other.sample_quantiles
+            
+            if len(s_self) > len(s_other):
+                raise ValueError("Sample size of `other` must be equal or " +
+                                 "larger than this `ProbPlot` instance")
```

Comment:
I'm just thinking it's probably good to make it consistent that  y-axis always using raw samples from `self` and x-axis could be interpolated samples if size of `other` is larger.